pom.xml

```diff
@@ -237,7 +237,7 @@
         <surefire.args>${surefire.args.base}</surefire.args>
 
         <commonsCollectionsVersion>3.2.2</commonsCollectionsVersion>
-        <groovyVersion>3.0.7</groovyVersion>
+        <groovyVersion>4.0.11</groovyVersion>
```

Comment:
Be aware that this property is also defined in the BOM:
https://github.com/mulesoft/mule-runtime-bom/blob/master/pom.xml#L55

I'm not sure if this change will end up taking precedence and potentially affecting the version of [this](https://github.com/mulesoft/mule-runtime-bom/blob/master/pom.xml#L153).